src/lang/de.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+const de = {
+  headings: {
+    contact: 'Kontakt',
+    experience: 'Erfahrung',
```

Comment:
I am not quite sure, but in my CV I used: 

Erfahrung -> Berufserfahrung
Fähigkeiten -> Qualifikationen

What do you think, might that more suitable?

Reply:
I'll change that. That were only my first thoughts about the translation.